cap1.js

```diff
@@ -0,0 +1,82 @@
+const hamburger = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  mobileMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.mobile-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  mobileMenu.classList.remove('active');
+}));
+
+const speakerInfo = [
```

Comment:
Great design @TiteLunda 🤝

- [ ] Kindly fix the text for the speakers' section. You may reduce the font size and remove the italicized text so that your design can replicate the one provided in the design.
- Your design
![your speakers](https://user-images.githubusercontent.com/63917725/235023183-8d4dfaeb-42e4-4a03-8b8f-712cc5b9b04c.PNG)
- Given design
![given speakers](https://user-images.githubusercontent.com/63917725/235023254-8f2f01ab-24d3-4d9f-a16e-e8ac832991c1.PNG)